editor/project_converter_3_to_4.cpp

```diff
@@ -2130,7 +2130,7 @@ bool ProjectConverter3To4::test_conversion(const RegExContainer &reg_container)
 	valid = valid & test_conversion_single_additional_builtin("set_cell_item(a, b)", "set_cell_item(a, b)", &ProjectConverter3To4::rename_gdscript_functions, "custom rename", reg_container, false);
 	valid = valid & test_conversion_single_additional_builtin("get_cell_item_orientation(a, b,c)", "get_cell_item_orientation(Vector3i(a,b,c))", &ProjectConverter3To4::rename_gdscript_functions, "custom rename", reg_container, false);
 	valid = valid & test_conversion_single_additional_builtin("get_cell_item(a, b,c)", "get_cell_item(Vector3i(a,b,c))", &ProjectConverter3To4::rename_gdscript_functions, "custom rename", reg_container, false);
-	valid = valid & test_conversion_single_additional_builtin("map_to_world(a, b,c)", "map_to_world(Vector3i(a,b,c))", &ProjectConverter3To4::rename_gdscript_functions, "custom rename", reg_container, false);
```

Comment:
The two name changes should be added to the list of renamed functions too. Otherwise I think this line will end up useless.

Reply:
Should be good now.

Reply:
Don't you have to add it to `gdscript_function_renames` too? I'd like @qarmin 's review on that as I am not really familiar with the project converter code.